docs/source/cheat_sheet.rst

```diff
@@ -106,7 +106,7 @@ Functions
                   body=None    # type: List[str]
                   ):
        # type: (...) -> bool
-        <code>
+       pass
```

Comment:
Whoops, just tested it and `pass` does typecheck :)
Maybe then `pass` is fine to keep.